synapse/cortex.py

```diff
@@ -693,6 +691,9 @@ def addRuntPropDel(self, prop, func):
         return ret
 
     async def runTagAdd(self, node, tag, valu):
+
+        await self.triggers.runTagAdd(node, tag)
```

Comment:
this means that `reindex --fire-handlers #someTag` would cause tag triggers to execute

Reply:
this would also apply to the node add handlers as well :|

Reply:
If we pushed these the trigger firing for tags up to the node; and onAdd/OnDel to the editatom, we would avoid having `--fire-handlers` fire triggers.

Reply:
Per discussion, we are deferring this until a large callback/architecture change.